source/chapters/user_interface.rst

```diff
@@ -712,35 +719,33 @@ Channel Faders and Level Meters
 Section Expansion Buttons
 -------------------------
 
-.. figure:: ../_static/Mixxx-200-Latenight-Top-Expansion-Buttons.png
+  .. figure:: ../_static/2.2/expansion-buttons-top.png
    :align: center
-   :width: 448px
-   :figwidth: 100%
+   :width: 430px
+   :figwidth: 90%
    :alt: The section expansion buttons
    :figclass: pretty-figures
 
    The section expansion buttons on top
 
-   .. figure:: ../_static/Mixxx-200-Latenight-Bottom-Expansion-Buttons.png
-      :align: center
-      :width: 448px
-      :figwidth: 100%
-      :alt: The section expansion buttons
-      :figclass: pretty-figures
+  .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
+    :align: center
+    :width: 245px
+    :figwidth: 90%
+    :alt: The section expansion buttons
+    :figclass: pretty-figures
 
-      The section expansion buttons on the bottom
+    The section expansion options under the Skin settings on the right.
 
-On top, and on the button of the user interface are the section expansion buttons.
+On top, and on the right side of the user interface are the section expansion buttons and options.
 
-If you click on either :guilabel:`MIC`, :guilabel:`SAMPLER`, or
-:guilabel:`VINYL` then you will enable control sections for interacting with:
+If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
 
-* :ref:`interface-mic`
+* :ref:`interface-effects`
 * :ref:`interface-sampler`
-* :ref:`interface-vc-mode`
+* :ref:`interface-mic`
 
 You can also use the specific :ref:`appendix-shortcuts`.
-
```

Comment:
Please revert this change. The missing newline makes the anchor below invalid, which is why the build error occurs:

```
/opt/build/repo/source/chapters/example_setups.rst:30:undefined label: interface-head-master (if the link has no caption the label must precede a section header)
```

Reply:
Oh. Thanks! I had missed that.